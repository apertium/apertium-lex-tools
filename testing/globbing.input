^blah<n>/bloop<n>/shoop<n>$ ^*hiya/*hiya$
^blah<n>/bloop<n>/shoop<n>$ ^hiya<n>/hiya<n>$
^blah<n>/bloop<n>/shoop<n>$ ^hiya<n><nom><sg>/hiya<n>$

^blah<n>/bloop<n>/shoop<n>$ ^hi<n><nom><sg>/hi<n>$
^blah<n>/bloop<n>/shoop<n>$ ^hi<n>/hiya<n>$

^blah<n>/bloop<n>/shoop<n>$ ^hi<ij><q>/hiya<n>$
^blah<n>/bloop<n>/shoop<n>$ ^hi<ij><n><q>/hiya<n>$

^blah<n>/bloop<n>/shoop<n>$ ^hi<v><itj>/hiya<n>$
^blah<n>/bloop<n>/shoop<n>$ ^hi<v><inf><itj>/hiya<n>$
^blah<n>/bloop<n>/shoop<n>$ ^hi<v><inf><pl><itj>/hiya<n>$
