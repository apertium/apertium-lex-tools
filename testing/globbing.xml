<lrx glob="star">
	<rules>
		<rule weight="1.0">
			<match lemma="blah" tags="n">
				<select lemma="bloop"/>
			</match>
			<match lemma="hiya" tags="*"/>
		</rule>
		<rule weight="1.0">
			<match lemma="blah" tags="n">
				<select lemma="bloop"/>
			</match>
			<match lemma="hi" tags="n.+"/>
		</rule>
		<rule weight="1.0">
			<match lemma="blah" tags="n">
				<select lemma="bloop"/>
			</match>
			<match lemma="hi" tags="ij.?.q"/>
		</rule>
		<rule weight="1.0">
			<match lemma="blah" tags="n">
				<select lemma="bloop"/>
			</match>
			<match lemma="hi" tags="v.*.itj"/>
		</rule>
		<rule weight="0.1">
			<match lemma="blah" tags="n">
				<select lemma="shoop"/>
			</match>
		</rule>
	</rules>
</lrx>
