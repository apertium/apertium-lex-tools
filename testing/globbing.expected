^blah<n>/bloop<n>$ ^*hiya/*hiya$
^blah<n>/bloop<n>$ ^hiya<n>/hiya<n>$
^blah<n>/bloop<n>$ ^hiya<n><nom><sg>/hiya<n>$

^blah<n>/bloop<n>$ ^hi<n><nom><sg>/hi<n>$
^blah<n>/shoop<n>$ ^hi<n>/hiya<n>$

^blah<n>/shoop<n>$ ^hi<ij><q>/hiya<n>$
^blah<n>/bloop<n>$ ^hi<ij><n><q>/hiya<n>$

^blah<n>/bloop<n>$ ^hi<v><itj>/hiya<n>$
^blah<n>/bloop<n>$ ^hi<v><inf><itj>/hiya<n>$
^blah<n>/bloop<n>$ ^hi<v><inf><pl><itj>/hiya<n>$
